<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderForm</class>
 <widget class="QWidget" name="OrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table #</string>
  </property>
  <widget class="QLineEdit" name="MenuItemEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Menu Item</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TableText">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Table #0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveOrderButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelOrderButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveItemButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>Remove Selected Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddItemButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Employeelabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee Name and Number</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Order Total</string>
   </property>
   <widget class="QLabel" name="subLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtotal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="taxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tax:</string>
    </property>
   </widget>
   <widget class="QLabel" name="grandTotalLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Grand Total:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubtotalText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TaxText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="GrantTotalText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="custSelectSpin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="custSelectLable">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer (0 for all)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CashOutButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash Out Order</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="custSpin">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="custLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer #</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantSpin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="empNumLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="tableNumLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="OrderTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="orderTab">
    <attribute name="title">
     <string>Current Order</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menuTab">
    <attribute name="title">
     <string>Menu</string>
    </attribute>
    <widget class="QListWidget" name="ItemListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
